### DISCUSSION TITLE
Flink SQL server CDC connector scans through all databases

### DISCUSSION THREAD

**Discussion by ennox108** [apache/flink-cdc#2524] [Q&A]
I am trying to run the Flink SQL server CDC connector on EMR. I have multiple databases in the same SQL server and they have certain tables that are common. When I try to read a particular table lets say A from database X into a Flink table the connector ends up scanning all the tables with name A under every database. I specified the property 'database-name' = 'X' during creation of the Flink table yet it scans all other databases. How do I avoid this behavior?

**Comment by GOODBOY008** [comment]
@ennox108 Fixed in master.

**Reply by ennox108** [reply]
@GOODBOY008 can you point me to the fix? We are using the flink-sql-connector-sqlserver-cdc-2.4.0.jar

**Reply by ennox108** [reply]
@GOODBOY008 i tried using 2.4.1 and it worked. However, I want to setup the connector on Flink 1.16 as I want to write the tables in Hudi format. And hudi works with flink 1.16 as per [https://hudi.apache.org/docs/flink-quick-start-guide/]
Is there any compatibility issue for flink-sql-connector-sqlserver-cdc-2.4.1 with Flink 1.16?
